### PR TITLE
[TKAI-3597] Add bug bounty to the footer

### PR DESCRIPTION
**Pull request by RTAndrew**

### CODE REVIEW

File: lib/locales/fr/frontend/navigation.json [hunk 1]
```diff
@@ -48,5 +48,6 @@
   "footer_menu_terms": "Conditions générales",
   "footer_menu_privacy": "Politique de confidentialité",
   "footer_description": "La plateforme pour Lancer et Rejoindre des Hackathons",
-  "copyright": "Tous droits réservés. Propulsé par"
+  "copyright": "Tous droits réservés. Propulsé par",
+  "bug_bounty": "Prime aux Bogues"
```
Comment (ickas) [inline lib/locales/fr/frontend/navigation.json:52]:
```suggestion
  "bug_bounty": "Programme de Bug Bounty"
```

File: lib/locales/es/frontend/navigation.json [hunk 2]
```diff
@@ -48,5 +48,6 @@
   "footer_menu_terms": "Términos y Condiciones",
   "footer_menu_privacy": "Política de Privacidad",
   "footer_description": "La plataforma para lanzar y unirse a hackathons",
-  "copyright": "Todos los derechos reservados. Potenciado por"
+  "copyright": "Todos los derechos reservados. Potenciado por",
+  "bug_bounty": "Recompensa de Bugs"
```
Comment (ickas) [inline lib/locales/es/frontend/navigation.json:52]:
```suggestion
  "bug_bounty": "Programa de Bug Bounty"
```

File: lib/locales/en/frontend/navigation.json [hunk 3]
```diff
@@ -48,5 +48,6 @@
   "footer_menu_terms": "Terms & Conditions",
   "footer_menu_privacy": "Privacy Policy",
   "footer_description": "The platform to Launch and Join Hackathons",
-  "copyright": "All rights reserved. Powered by"
+  "copyright": "All rights reserved. Powered by",
+  "bug_bounty": "Bug Bounty"
```
Comment (ickas) [inline lib/locales/en/frontend/navigation.json:52]:
```suggestion
  "bug_bounty": "Bug Bounty Program"
```

File: lib/locales/br/frontend/navigation.json [hunk 4]
```diff
@@ -48,5 +48,6 @@
   "footer_menu_terms": "Termos e Condições",
   "footer_menu_privacy": "Política de Privacidade",
   "footer_description": "A plataforma para Organizar e Participar de Hackathons",
-  "copyright": "Todos os direitos reservados. Desenvolvido por"
+  "copyright": "Todos os direitos reservados. Desenvolvido por",
+  "bug_bounty": "Recompensa de Bugs"
```
Comment (ickas) [inline lib/locales/br/frontend/navigation.json:52]:
```suggestion
  "bug_bounty": "Programa de Bug Bounty"
```

File: lib/locales/fr/frontend/navigation.json (hunk 1 above)
Comment (RTAndrew) [reply lib/locales/fr/frontend/navigation.json:52]: A quick search says that `Prime aux Bogues` is more common. Would you rather continue with your suggestion?

image

**Review by ickas — APPROVED** [review]

**Pull request merged**